.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: "🚀 Feature Request"
+about: I have a suggestion
+labels: type/enhancement
+---
+
+## Feature Request
+
+**Is your feature request related to a problem? Please describe:**
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+
+**Describe the feature you'd like:**
+<!-- A clear and concise description of what you want to happen. -->
+
+**Describe alternatives you've considered:**
+<!-- A clear and concise description of any alternative solutions or features you've considered. -->
+
+**Teachability, Documentation, Adoption, Migration Strategy:**
+<!-- If you can, explain some scenarios how users might use this, situations it would be helpful in. Any API designs, mockups, or diagrams are also he
```

Comment:
```suggestion
<!-- If you can, explain some scenarios on how users might use this, situations it would be helpful in. Any API designs, mockups, or diagrams are also he
```

Reply:
I'm confused about "or diagrams are also he"

Reply:
me too

Reply:
Actually, This is a copy from tidb repo :(